auto_tune_vllm/cli/main.py

```
@@ -0,0 +1,856 @@
+    verbose: bool = typer.Option(False, "--verbose", "-v", help="Verbose logging"),
+):
+    """Clear study data (trials, parameters, and optionally logs)."""
+    setup_logging(verbose)
+    
+    console.print(f"[bold yellow]Preparing to clear study: {study_name}[/bold yellow]")
+    
+    try:
+        # Verify database connections
+        if not verify_database_connection(database_url):
+            console.print(f"[bold red]Error: Cannot connect to database: {database_url}[/bold red]")
+            raise typer.Exit(1)
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)